src/main/resources/edu/wpi/first/pathui/style.css

```diff
@@ -2,19 +2,50 @@
     -fx-background-color: #101824;
 }
 
-Circle {
-    -fx-fill: orange;
+:subchild0 {
```

Comment:
This should be

```css
.tree-cell:subchild0 {
    -fx-background-color: CHART_COLOR_2;
}

CubicCurve:subchild0 {
    -fx-stroke: CHART_COLOR_2;
}
```

I recommend setting a custom style class for the splines (maybe `path`?) instead of using the class selector `CubicCurve`, as well as creating your own variables for the colors for better naming.